src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9401 @@
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_parameters[0] = Parameter::UNIT;
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_parameters[1] = Parameter::INT;
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_uiStrings[0] = "Warehouse named ";
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_uiStrings[1] = " is set to having ";
+    m_actionTemplates[ScriptAction::WAREHOUSE_SET_VALUE].m_uiStrings[2] = " dollars worth of boxes. ";
+
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_internalName = "SOUND_DISABLE_TYPE";
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_uiName = "Multimedia_/ Sound Events -- disable type.";
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SOUND_DISABLE_TYPE].m_uiStrings[1] = " is disabled.";
+
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_internalName = "SOUND_ENABLE_TYPE";
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_uiName = "Multimedia_/ Sound Events -- enable type.";
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_TYPE].m_uiStrings[1] = " is enabled.";
+
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_internalName = "SOUND_REMOVE_TYPE";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_uiName = "Multimedia_/ Sound Events -- remove type.";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_numParameters = 1;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_parameters[0] = Parameter::SOUND;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_numUiStrings = 2;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_uiStrings[0] = " ";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_TYPE].m_uiStrings[1] = " is removed.";
+
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_internalName = "SOUND_REMOVE_ALL_DISABLED";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_uiName =
+        "Multimedia_/ Sound Events -- remove all disabled.";
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_numUiStrings = 1;
+    m_actionTemplates[ScriptAction::SOUND_REMOVE_ALL_DISABLED].m_uiStrings[0] = "Remove all disabled sound events.";
+
+    m_actionTemplates[ScriptAction::SOUND_ENABLE_ALL].m_internalName = "SOUND_ENABLE_ALL";
```

Comment:
Does not set m_numParameters

Reply:
Original code does not set it (and leaves it at the default of 0)